embc-app/SeedData/Communities.json

```diff
@@ -1,14 +1,3788 @@
+        "RegionalDistrict": {
+            "Name": "Cariboo Regional District"
+        }
+    },
+    {
+        "Name": "Nimpo Lake",
+        "RegionalDistrict": {
+            "Name": "Cariboo Regional District"
+        }
+    },
+    {
+        "Name": "Quesnel",
+        "RegionalDistrict": {
+            "Name": "Cariboo Regional District"
+        }
+    },
+    {
+        "Name": "Quesnel Forks",
+        "RegionalDistrict": {
+            "Name": "Cariboo Regional District"
+        }
+    },
+    {
+        "Name": "Red Bluff",
+        "RegionalDistrict": {
+            "Name": "Cariboo Regional District"
+        }
+    },
+    {
+        "Name": "Redstone",
+        "RegionalDistrict": {
+            "Name": "Cariboo Regional District"
+        }
+    },
+    {
+        "Name": "Kimsquit",
+        "RegionalDistrict": {
+            "Name": "Central Coast Regional District"
+        }
+    },
+    {
+        "Name": "Margaret Bay",
+        "RegionalDistrict": {
+            "Name": "Central Coast Regional District"
+        }
+    },
+    {
+        "Name": "Namu",
+        "RegionalDistrict": {
+            "Name": "Central Coast Regional District"
+        }
+    },
+    {
+        "Name": "Ocean Falls",
+        "RegionalDistrict": {
+            "Name": "Central Coast Regional District"
+        }
+    },
+    {
+        "Name": "Rivers Inlet",
+        "RegionalDistrict": {
+            "Name": "Central Coast Regional District"
+        }
+    },
+    {
+        "Name": "Shearwater",
+        "RegionalDistrict": {
+            "Name": "Central Coast Regional District"
+        }
+    },
+    {
+        "Name": "Stuie",
+        "RegionalDistrict": {
+            "Name": "Central Coast Regional District"
+        }
+    },
+    {
+        "Name": "Tallheo",
+        "RegionalDistrict": {
+            "Name": "Central Coast Regional District"
+        }
+    },
+    {
+        "Name": "Wadhams",
+        "RegionalDistrict": {
+            "Name": "Central Coast Regional District"
+        }
+    },
+    {
+        "Name": "Aleza Lake",
+        "RegionalDistrict": {
+            "Name": "Regional District of Fraser - Fort George"
+        }
+    },
+    {
+        "Name": "Baldy Hughes",
+        "RegionalDistrict": {
+            "Name": "Regional District of Fraser - Fort George"
+        }
+    },
+    {
+        "Name": "Bear Lake",
+        "RegionalDistrict": {
+            "Name": "Regional District of Fraser - Fort George"
+        }
+    },
+    {
+        "Name": "Bednesti",
+        "RegionalDistrict": {
+            "Name": "Regional District of Fraser - Fort George"
+        }
+    },
+    {
+        "Name": "Bend",
+        "RegionalDistrict": {
+            "Name": "Regional District of Fraser - Fort George"
+        }
+    },
+    {
+        "Name": "Crescent Spur",
+        "RegionalDistrict": {
+            "Name": "Regional District of Fraser - Fort George"
+        }
+    },
+    
```

Comment:
@CodingVelocista could you please verify there are no "artifacts" in the data like the one "?" above

Reply:
Not sure what you are referring to in the above.

Reply:
![image](https://user-images.githubusercontent.com/24322224/54726770-c4dfe900-4b31-11e9-91bd-0811a566ce92.png)

Do you see a question mark as I've pasted here? It might be a UTF char that's not displayed properly on GitHub, maybe?